authentication/auth.go

```diff
@@ -793,8 +780,8 @@ func GetDefaultValues() map[string]string {
 	defaults["TEST_QUESTIONS"] = "F"
 	defaults["sds_dataset_id"] = sdsDatasetId.String()
 	defaults["survey_id"] = "123"
-	defaults["WINDOW_START_DATE"] = "2023-03-01"
-	defaults["WINDOW_CLOSE_DATE"] = "2023-03-31"
+	defaults["WINDOW_START_DATE"] = "2016-05-01"
```

Comment:
Why was this date changed?

Reply:
I changed this date for it to be consistent with the `ref_p_start_date` and `ref_p_end_date` as Mebin and I discussed this too.